app/lib/signup/account_manager.rb

```diff
@@ -96,39 +96,21 @@ def selected=(plans)
     end
 
     delegate :[], to: :plans
-    delegate :to_a, to: :values
```

Comment:
Because there is already another `to_a` defined, and this one wouldn't even work because that `values` is not defined :joy: